### DISCUSSION TITLE
Help with redirecting YOURLS

### DISCUSSION THREAD

**Discussion by jardaoravec** [YOURLS/YOURLS#3822] [General]
Hello,

I have a question about using **YOURLS** for link shortening in a specific setup.

I would like to shorten links for a domain (let’s call it domain1.com), but I do not have FTP access to that domain,  but i can add DNS dns records to domain1.com.  However, I have full FTP access to a second domain (domain2.com), where I can install YOURLS.

Is there any way I can configure YOURLS to shorten links for domain1.com, given that I don’t have FTP access to it? I am able to set up redirects (e.g., through DNS or other configuration options) on domain1.com, but I cannot directly upload or manage files via FTP there.

Thank you in advance for any advice or solutions!

**Comment by wrtz008** [comment]
You can only add Yourls redirections on the domain that you have access to.
i.e. domain2.com you have access and you install Yourl, then you can add the short Yourl slug to point to your domain1.com

But on domain1.com you can't add a Yourl because you don't have Yourl installed there -- unless I am missing the point here?
